modules/codemod/lib/v9/spec/compoundToast.spec.ts

```diff
@@ -135,4 +135,35 @@ describe('Toast', () => {
 
     expectTransform(input, expected);
   });
+
+  it('should not restructure Toast usage if it is already compound', () => {
+    const input = stripIndent`
+      import {Toast} from '@workday/canvas-kit-labs-react/toast';
```

Comment:
If you change order, make sure you change import path to `@workday/canvas-kit-labs-react/toast` for every other test

Reply:
nvm